### PR TITLE
[BTFS-1477] storage new endpoints mock up

### PR DESCRIPTION
**Pull request by wxue**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* mockup 5 new endpoints with dummy statics return
    * storage stat info, sync
    * storage contracts stat, sync, list

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior?** (You can also refer to a JIRA ticket here)

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **What dependencies / modules need to be updated as pre-requisites?** (Include relevant PR links here)
ref: https://github.com/TRON-US/go-btfs-common/pull/78

go-btfs-common's PR need to merge first(then update mod)

* **Description of changes**

---

* **Please check if the PR fulfills these requirements**
- [x] The subject of this PR contains a JIRA ticket BTFS-xxxx (N/A for community PRs)
- [x] PR description and commit messages are [good and meaningful](https://chris.beams.io/posts/git-commit/)
- [x] Unit tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **Please make sure the following procedures have been applied before requesting reviewers**
- [x] Code changes closely follow [Go Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments)
- [x] Code has been *rebased* against latest master (`git merge` not recommended, unless you know what you are doing)
- [x] Code changes have run through `go fmt`
- [x] Code changes have run through `go mod tidy`
- [x] All unit tests passed locally (`make test_go_test`)

### CODE REVIEW

**Comment by wxue** [comment]
ref: https://github.com/TRON-US/go-btfs-common/pull/78

go-btfs-common's PR need to merge first(then update mod)

**Comment by wxue** [comment]
Static Dummy APIs are up:
http://localhost:5001/api/v1/storage/contracts/stat

**Comment by wxue** [comment]
@taiyangc  ready for review now, thanks

File: core/commands/commands_test.go [hunk 1]
```diff
@@ -233,6 +233,13 @@ func TestCommands(t *testing.T) {
 		"/storage/challenge",
 		"/storage/challenge/request",
 		"/storage/challenge/response",
+		"/storage/stat",
+		"/storage/stat/info",
+		"/storage/stat/sync",
```
Comment (taiyangc) [inline core/commands/commands_test.go]: To follow this convention, the command should be `stats`: https://docs-beta.ipfs.io/reference/cli/#ipfs-stats

File: core/commands/storage.go [hunk 2]
```diff
@@ -1528,3 +1530,165 @@ the challenge request back to the caller.`,
...
+
+// Storage Stats
+//
+// Includes sub-commands: info, sync
+var storageStatsCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "Get node storage stat in the network.",
+		ShortDescription: `
+btfs storage stat sync <>
+btfs storage stat info <>`,
```
Comment (taiyangc) [inline core/commands/storage.go]: no need for this, sub-commands format are auto-populated.

File: core/commands/commands_test.go (hunk 1 above)
Comment (wxue) [reply core/commands/commands_test.go]: got it, will update code and design doc

File: core/commands/storage.go (hunk 2 above)
Comment (wxue) [reply core/commands/storage.go]: ok, but fyi, the other commands have these coded in

**Pull request merged**